documentation/modules/exploit/linux/http/denyall_waf_exec.md

```diff
@@ -0,0 +1,37 @@
+## Vulnerable Application
+
+This module exploits the command injection vulnerability of DenyAll Web Application Firewall. Unauthenticated user can execute a terminal command under the context of the web server user.
+
+**Vulnerable Application Installation Steps**
```

Comment:
For consistency I think you can remove this bold area and just let it all blend in to one.